build/release-notes/avalanchego.md

```diff
@@ -2,6 +2,20 @@
 
 {% page-ref page="../tutorials/nodes-and-staking/upgrade-your-avalanchego-node.md" %}
 
+## v1.4.12 \([View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.4.12)\)
+
+This update is backwards compatible. It is optional, but encouraged.
+
```

Comment:
We also updated the shared memory interface (Replaces the `Put` and `Remove`) calls with `Apply` with was done with an eye on future atomic tx optimizations

Reply:
We also updated the database benchmarks in this PR